addons/mail/models/discuss/discuss_channel.py

```diff
@@ -796,12 +796,14 @@ def _channel_info(self):
                 # exclude chat and other small channels from this optimization because they are
                 # assumed to be smaller and it's important to know the member list for them
                 info['channelMembers'] = [('ADD', list(members_by_channel[channel]._discuss_channel_member_format().values()))]
+                # sort seen infos by type (partner first, then guest) then by id
```

Comment:
why not just by member id?

Reply:
We're already sorting them, I believe it's easier to read when they are split.